src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js

```diff
@@ -81,8 +81,8 @@ function DateOfBirthPage({translate, privatePersonalDetails}) {
                         inputID="dob"
                         label={translate('common.date')}
                         defaultValue={privatePersonalDetails.dob || ''}
-                        minDate={moment().subtract(CONST.DATE_BIRTH.MAX_AGE, 'years').toDate()}
-                        maxDate={moment().subtract(CONST.DATE_BIRTH.MIN_AGE, 'years').toDate()}
+                        minDate={subYears(new Date(), CONST.DATE_BIRTH.MAX_AGE)}
```

Comment:
if this returns an object might be better to wrap it using useMemo

Reply:
what do you mean?
I changed only moment to date-fns
Its out of scope of this issue